superset_patchup/oauth.py

```diff
@@ -80,6 +76,33 @@ def login(self, provider=None, register=None):
             flash(as_unicode(self.invalid_login_message), "warning")
             return redirect(redirect_url)
 
+    @expose("/oauth-init/<provider>")
+    def login_init(self, provider=None):
```

Comment:
@dserkowski could you add some docs on how this is to be used?  If you prefer to add it as a comment here as opposed to editing the README that is fine, I will add it to the README later.  I right after that we'll merge and release this.  Looks like a really good change, thank you!